cmd/bosun/conf/notify.go

```diff
@@ -12,8 +12,18 @@ import (
 
 	"bosun.org/_third_party/github.com/jordan-wright/email"
 	"bosun.org/collect"
+	"bosun.org/metadata"
 )
 
+func init() {
+	metadata.AddMetricMeta(
+		"bosun.email.sent", metadata.Counter, metadata.PerSecond,
+		"Counter of the number of times Bosun has sent an email notification.")
```

Comment:
"The number of email notifications sent by Bosun." , "The number of email notifications that Bosun failed to send."